.gitlab-ci.yml

```diff
@@ -38,17 +38,12 @@ variables:
   DEVICEMONITOR_REV: "master"
   MONITOR_CLIENT_REV: "master"
   REPORTING_REV: "master"
+  MENDER_GATEWAY_REV: "master"
```

Comment:
I'm confused. Are we removing it or not?

Reply:
We are removing it from [here](https://github.com/mendersoftware/mender-qa/pull/572/files#diff-3d8f72bfc9d467ad4827724f6630b201f58ad1211054f34778da72fb20a6a3b4L5). In other words, we don't have `MENDER_GATEWAY_REV` as an especial case that needs to be also defined in CI/CD variables but just treat it as every other `_REV` variable. That is what I tried to express in the commit message :sweat_smile: